iOSClient/Settings/CCManageCameraUpload.m

```diff
@@ -62,6 +62,20 @@ - (void)initializeForm
 
     form = [XLFormDescriptor formDescriptorWithTitle:NSLocalizedString(@"_uploading_from_camera_", nil)];
     
+    // Request permission for camera roll access
```

Comment:
Agreed. But I do think this pull request contributes to a relevant UX improvement. :)